src/utils/validation/registration/fundingValidation.ts

```diff
@@ -9,6 +9,9 @@ const fundingErrorMessage = {
   fundingProjectRequired: i18n.t('feedback.validation.is_required', {
     field: i18n.t('registration.description.funding.funding_name'),
   }),
+  fundingNameOrIdentifierRequired: i18n.t('feedback.validation.is_required', {
+    field: `${i18n.t('registration.description.funding.funding_name')} ${i18n.t('common.or').toLowerCase()} ${i18n.t('registration.description.funding.funding_id')}`,
```

Comment:
Ikke fan av denne måten å slå sammen 3 tekster på, men det gjør at vi slipper å legge inn en ny oversettelse for dette da, og vi vet at navn på felter blir riktig. Derfor tenker jeg at det kan være OK akkurat her, selv om det har sine drawbacks..